Jenkinsfile

```diff
@@ -1,5 +1,7 @@
+@Library('pipeline-library@pull/499/head') _
+
 /*
  See the documentation for more options:
  https://github.com/jenkins-infra/pipeline-library/
 */
-buildPlugin(useContainerAgent: true, platforms: ['linux'], jdkVersions: ['8','11','17'])
+buildPlugin(useContainerAgent: false, platforms: ['linux', 'windows'], artifactCachingProxyEnabled: true, jdkVersions: ['8','11','17'])
```

Comment:
`useContainerAgent` also tested?

Reply:
@jglick successfully tested [with a replay (Linux only for now)](https://ci.jenkins.io/job/Plugins/job/jenkins-infra-test-plugin/job/PR-44/128):
```
@Library('pipeline-library@pull/499/head') _

/*
 See the documentation for more options:
 https://github.com/jenkins-infra/pipeline-library/
*/
buildPlugin(useContainerAgent: true, platforms: ['linux'], artifactCachingProxyEnabled: true, jdkVersions: ['11'])

```

Reply:
Testing my new PR on both, looks good.